From d5dfd2c1f85372553c42496d74df56c823dd215e Mon Sep 17 00:00:00 2001
From: Sanyog Kale <sanyog.r.kale@intel.com>
Date: Fri, 6 Jan 2017 16:12:13 +0530
Subject: [PATCH 0731/2367] ASoC: Intel: Kconfig changes for SoundWire
 aggregation support

This patch adds SoundWire aggregation config.

Change-Id: I7e5ab3d7079454dfde16b98caec818dd21e47c99
Signed-off-by: Sanyog Kale <sanyog.r.kale@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/12938
Reviewed-by: Jayanti, Satya Charitardha <satya.charitardha.jayanti@intel.com>
Reviewed-by: Nc, Shreyas <shreyas.nc@intel.com>
Reviewed-by: Singh, Guneshwor O <guneshwor.o.singh@intel.com>
Reviewed-by: Kp, Jeeja <jeeja.kp@intel.com>
Reviewed-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Tested-by: Avati, Santosh Kumar <santosh.kumar.avati@intel.com>
---
 sound/soc/intel/Kconfig | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/sound/soc/intel/Kconfig b/sound/soc/intel/Kconfig
index a1ad9d2..81cdfca 100644
--- a/sound/soc/intel/Kconfig
+++ b/sound/soc/intel/Kconfig
@@ -357,9 +357,6 @@ config SND_SOC_INTEL_CNL_SVFPGA_MACH
 	select SND_SOC_INTEL_SKYLAKE
 	select SND_SOC_DMIC
         help
-	This adds support for ASoC machine driver for cnl and SVFPGA. This will
-	create an alsa sound card.  Say Y if you have such a device If unsure
-	select "N".
 
 config SND_SOC_INTEL_CNL_RT274_MACH
         tristate "ASoC Audio driver for CNL with RT274 in I2S Mode"
@@ -372,3 +369,8 @@ config SND_SOC_INTEL_CNL_RT274_MACH
 	This adds support for ASoC machine driver for CNL and codec RT274. This
 	will create an alsa sound card.  Say Y if you have such a device If
 	unsure select "N".
+
+config SND_SOC_SDW_AGGM1M2
+	bool "Enable SoundWire aggregation on Master 1 and Master 2"
+	help
+	  Say Y to enable SoundWire aggregation on Master1 and Master2.
-- 
2.7.4

